app/src/main/java/org/systers/mentorship/remote/TokenAuthenticator.kt

```diff
@@ -17,6 +17,11 @@ class TokenAuthenticator: Authenticator{
 
     override fun authenticate(route: Route, response: Response): Request? {
         if (response.code() == 401) {
+
+            if (response.request().url().encodedPath() == "/login" ) {
```

Comment:
Awesome job, I was just wondering if we should add the `"/login"` text as a constant and then add it as the first condition to make it more readable.

```
private const val LOGIN_PATH = "/login"
```
```suggestion
            if (LOGIN_PATH == response.request().url().encodedPath()) {
```

Reply:
I am not sure we'll need the login path anywhere else .